src/flb_log.c

```diff
@@ -454,6 +478,78 @@ int flb_log_set_file(struct flb_config *config, char *out)
     return 0;
 }
 
+
+/*
+ * Create and register cmetrics for the runtime logger.
+ * The caller must free the returned struct using flb_log_metrics_destroy.
+ */
+struct flb_log_metrics *flb_log_metrics_create()
+{
+    struct flb_log_metrics *metrics;
+    int i;
+    const char *message_type_str;
+    uint64_t ts;
+    int ret;
+
+    metrics = flb_calloc(1, sizeof(struct flb_log_metrics));
+    if (metrics == NULL) {
+        flb_errno();
+        return NULL;
+    }
+
+    metrics->cmt = cmt_create();
+    if (!metrics->cmt) {
```

Comment:
Please compare explicitly against `NULL` here and in any other places where it's not done in that way.

Reply:
Done.